ledger/participant-integration-api/src/main/scala/platform/apiserver/services/admin/ApiPackageManagementService.scala

```diff
@@ -43,6 +44,7 @@ private[apiserver] final class ApiPackageManagementService private (
     managementServiceTimeout: Duration,
     engine: Engine,
     darReader: GenDarReader[Archive],
+    submissionIdGenerator: String => v1.SubmissionId,
```

Comment:
We already import `SubmissionId` directly.

```suggestion
    submissionIdGenerator: String => SubmissionId,
```